### DISCUSSION TITLE
An example for any Symbol/sign on the chart

### DISCUSSION THREAD

**Discussion by AlexStox** [highfestiva/finplot#290] [Ideas]
Hi Jonas, I've found only this example (at least among those I can understand :) )
```
fplt.plot(x, y)
fplt.plot(x[mask], y[mask], style="--")
```
I can use this kind of line to mark some bars on the chart, i.e something happened. 
I also found `fplt.add_line(), fplt.add_text()` but did not understand how to use them. 

Could you please provide an example (which you can also add to the list of examples) on how to add any symbol (string) for specific bars on the chart? 

E.g. I have a column in the dataframe which I can use as a mask: 
```
df = pd.DataFrame({"open": [100, 105, 95,  99,  92],
                   "high": [110, 120, 115, 110, 115],
                   "low":  [92,  90,  85, 70,  81],
                  "close": [105, 95,  99, 92,  85],
                   "mask": [True, False, False, True, True]},
                  index=pd.date_range("2020-01-01", "2020-01-05"))
```
so how to use it to place a symbol (^) or string ('up') on the chart for every bar with mask='True'? 

Maybe there are easier ways to do it. E.g. I also saw finplot.show() recognizes df.plot(), but I didn't find any solution except for the 'scatter' plot for my purpose.

**Comment by highfestiva** [comment]
```python
df.loc[df['mask'], 'markers'] = df.low
fplt.plot(df.markers, style='^')
```

**Reply by AlexStox** [reply]
I tested it and found that not so many symbols are allowed. Is there any list of allowed symbols to use in style=... ? Or is there any other possibility to plot text/symbol above/under a specific bar?

**Reply by AlexStox** [reply]
I found available symbols in the QT library. 
C:\YOUR_USER\AppData\Local\Programs\Python\Python39\Lib\site-packages\pyqtgraph\graphicsItems\ScatterPlotItem.py
name_list = ['o', 's', 't', 't1', 't2', 't3', 'd', '+', 'x', 'p', 'h', 'star', 'arrow_up', 'arrow_right', 'arrow_down', 'arrow_left', 'crosshair']

However, it would be great if you suggest how to type/plot any word/symbol above/under a specific bar.
